app/konnect/runtime-manager/gateway-runtime-kubernetes.md

```diff
@@ -0,0 +1,168 @@
+
+1. Create a namespace:
+    ```sh
+    $ kubectl create namespace kong
+    ```
+
+2. Add the Kong charts repository:
+    ```bash
+    $ helm repo add kong https://charts.konghq.com
+    ```
+
+3. Update Helm:
+    ```bash
+    $ helm repo update
+    ```
+
+## Generate certificates
+
+{% include /md/konnect/runtime-certs.md %}
+
+## Configure the runtime
+
+Next, configure a {{site.base_gateway}} instance using the certificate, the
+private key, and the remaining configuration details on the
+**Configure Runtime** page.
+
+### Configure secrets
+
+Store the certificates and key you generated through the Runtime Manager in
+Kubernetes secrets.
+
+1. Create a `tls` secret using the `cluster.cert` and `cluster.key` files
+you saved earlier:
+
+    ```bash
+    $ kubectl create secret tls kong-cluster-cert -n kong \
+      --cert=/<path-to-file>/cluster.crt \
+      --key=/<path-to-file>/cluster.key
+    ```
+
+2. Create a generic secret for the `ca.crt` file:
+
+    ```bash
+    $ kubectl create secret generic kong-cluster-ca -n kong \
+      --from-file=ca.crt=/<path-to-file>/ca.crt
+    ```
+
+### Write configuration
+
+1. Back in {{site.konnect_short_name}}, copy the
+codeblock from the **Configuration Parameters** section.
+
+2. Create a `values.yaml` file. Remove the `KONG_` prefix from the parameters
+in the sample codeblock and add the following parameters to the file.
+
+    Make sure to replace the values in `cluster_cert`, `cluster_cert_key`,
+    and `cluster_ca_cert` with references to the secrets you created earlier:
+
+    ```yaml
+    image:
+      repository: kong-docker-kong-gateway-docker.bintray.io/kong-enterprise-edition
+      tag: "{{site.data.kong_latest_ee.version}}-alpine"
+
+    secretVolumes:
+    - kong-cluster-cert
+    - kong-cluster-ca
+
+    admin:
+      enabled: false
+
+    env:
+      role: data_plane
+      database: "off"
+      anonymous_reports: off
+      vitals_ttl_days: 732
+      cluster_mtls: pki
+      cluster_control_plane: <example.cp.konnect.foo>:443
+      cluster_server_name: <kong-cpoutlet-example.service>
+      cluster_telemetry_endpoint: <example.tp.konnect.foo>:443
+      cluster_telemetry_server_name: <kong-telemetry-example.service>
+      cluster_ca_cert: /etc/secrets/kong-cluster-ca/ca.crt
+      cluster_cert: /etc/secrets/kong-cluster-cert/tls.crt
+      cluster_cert_key: /etc/secrets/kong-cluster-cert/tls.key
+      lua_ssl_trusted_certificate: /etc/secrets/kong-cluster-ca/ca.crt
+
+    ingressController:
+      enabled: false
+      installCRDs: false
+    ```
+
+    See [Parameters](/konnect/runtime-manager/runtime-parameter-reference) for
+    descriptions and the matching fields in {{site.konnect_short_name}}.
+
+6. Apply the `values.yaml`:
+
+    ```bash
+    $ helm install my-kong kong/kong -n kong \
+      --values ./values.yaml
+    ```
+
+7. Clean up:
+    ```bash
+    $ kubectl delete jobs -n kong --all
```

Comment:
this could use some explanation about why this might be used

Reply:
Good question - pulled it from an [existing installation topic](https://docs.konghq.com/enterprise/2.3.x/deployment/installation/kong-on-kubernetes/#step-8-finalize-configuration-and-verify-installation). I'm guessing the answer is probably "good practice"?

Reply:
My understanding is the doc is targeting a user who already uses k8s. I'm no expert just trying to be helpful, but it seems to me k8s good practice is out of scope for this doc. Also as far as I can tell there aren't any jobs to run for a data plane, a typical job would be a db migration which happens at the control plane.